bundle/webpack.base.js

```diff
@@ -0,0 +1,56 @@
+const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
+const mode = process.env.NODE_ENV || 'development';
+
+// This is main configuration object.
+// Here you write different options and tell Webpack what to do
+module.exports = {
+
+  // Default mode for Webpack is production.
+  // Depending on mode Webpack will apply different things
+  // on final bundle.
+  mode,
+
+  entry: './src/index.ts',
+
+  optimization: { usedExports: true },
+
+  externals: {
+    '@luma.gl/core': {
```

Comment:
I see the `deck` case, but why do we need to set `luma` as externals? what's the use case?

Reply:
In fact, I am gonna add all `@deck.gl/**` modules to external dependencies as we are loading deck.gl via HTML script.

`luma.gl` is bundled within `@deck.gl/core`, so there's no need of including it twice, and it throws if you have two different versions imported.